lnurl.py

```diff
@@ -75,14 +75,23 @@ async def api_scan(p, c, request: Request, external_id: str):
     if hits_amount > card.daily_limit:
         return {"status": "ERROR", "reason": "Max daily limit spent."}
     hit = await create_hit(card.id, ip, agent, card.counter, ctr_int)
-    lnurlpay = lnurl_encode(str(request.url_for("boltcards.lnurlp_response", hit_id=hit.id)))
+
+    #the raw lnurl
+    lnurlpay_raw = str(request.url_for("boltcards.lnurlp_response", hit_id=hit.id))
+    #bech32 encoded lnurl
+    lnurlpay_bech32 = lnurl_encode(lnurlpay_raw)
+    #create a lud17 lnurlp to support lud19, add to payLink field of the withdrawRequest
```

Comment:
nitpick: style - one space after `#`
```suggestion
    # create a lud17 lnurlp to support lud19, add to payLink field of the withdrawRequest
```